src/containers/Bookings.js

```diff
@@ -26,28 +41,46 @@ export default class Bookings extends Component {
 
   sortBy(key) {
     this.setState({
-      result: FakeBookings.sort((a, b) => (
+      result: this.state.FakeBookings.sort((a, b) => (
+        this.state.direction[key] === 'asc'
+          ? (a[key].toLowerCase()) < (b[key].toLowerCase())
+          : (a[key].toLowerCase()) > (b[key].toLowerCase())
+      )),
+      direction: {
+        [key]: this.state.direction[key] === 'asc'
+          ? 'desc'
+          : 'asc'
+      },
+      text: this.state.text
+    })
+  }
+
+  sortByNumber(key) {
```

Comment:
I quite like your solution 👍 It's very explicit, which is great. There is an alternative where you just have 1 `sort` method and it checks whether the field that you're sorting by is a number or not. This would reduce some of the duplicated code that you have here.

But I'm not saying that is a "more correct" answer. There's a tradeoff here between the explicit-ness of your solution and the more complicated but less duplicated solution that I'm suggesting. You might not appreciate this but this is great!! We're having the kind of conversation that me & my colleagues have 👍